tests/api/test_runtime_event_ledger.py

```
@@ -0,0 +1,175 @@
+
+from penguin.system.runtime_event_ledger import (
+    RuntimeEventLedger,
+    RuntimeEventLedgerPolicy,
+)
+from penguin.system.runtime_events import (
+    build_runtime_event,
+    reset_runtime_event_sequences,
+)
+
+
+def _ledger(tmp_path: Path, **policy_kwargs) -> RuntimeEventLedger:
```

Ruff (0.15.20)

[warning] Missing type annotation for `**policy_kwargs`

(ANN003)